web/src/tippyjs.ts

```diff
@@ -520,4 +520,23 @@ export function initialize(): void {
         placement: "bottom",
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: "#move_topic_to_stream_widget_wrapper",
+        onShow(instance) {
+            if ($("#move_topic_to_stream_widget").prop("disabled")) {
+                instance.setContent(
+                    $t({
+                        defaultMessage:
+                            "You do not have permission to move messages to another stream in this organization.",
+                    }),
+                );
+                return undefined;
+            }
+            instance.destroy();
+            return false;
+        },
+        delay: INSTANT_HOVER_DELAY,
```

Comment:
I guess there is no need for this "delay" argument here, since it is set to `INSTANT_HOVER_DELAY` by default.

Reply:
Removed the "delay" argument.